components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/keystore/service/KeyStoreAdminServiceImpl.java

```diff
@@ -38,9 +41,14 @@ public KeyStoreData[] getKeyStores() throws SecurityConfigException {
     @Override
     public void addKeyStore(String fileData, String filename, String password, String provider,
                             String type, String pvtkeyPass) throws SecurityConfigException {
-        KeyStoreAdmin admin = new KeyStoreAdmin(CarbonContext.getThreadLocalCarbonContext().getTenantId(),
-                getGovernanceSystemRegistry());
-        admin.addKeyStore(fileData, filename, password, provider, type, pvtkeyPass);
+
```

Comment:
Shall we not change the KeyStoreAdmin class usages since we are planning to introduce a new constructor by removing the registry initializations.